controllers/argocd/rolebinding.go

```diff
@@ -19,6 +20,32 @@ import (
 	"github.com/argoproj-labs/argocd-operator/controllers/argoutil"
 )
 
+// truncates a string to maxLabelLength characters and adds a hash suffix to ensure uniqueness
```

Comment:
nit pick
// truncateWithHash truncates a string to a maximum of 63 characters and adds a hash suffix to ensure uniqueness